src/components/availability-panel/helper-functions/AssignTask.ts

```diff
@@ -0,0 +1,31 @@
+import axios, { AxiosRequestConfig } from 'axios';
+
+type Props = {
+  authToken: string;
+  taskId: string;
+  assignee: string;
+};
+
+const AssignTask = async ({ authToken, taskId, assignee }: Props) => {
```

Comment:
This is not a component, it's a `function` it should accept normal arguments not the `Props`.
```
const AssignTask = async (authToken: string, taskId: string, assignee: string) => {
    // ...
};
```